package.json

```diff
@@ -34,7 +34,7 @@
     "install-playwright": "npx playwright install",
     "security_audit": "npx audit-ci --config audit-ci.json",
     "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open --e2e --browser chrome",
+    "int-test-ui": "cypress open --e2e --browser electron",
```

Comment:
Is this intentional?  It seems a big step as I assume chrome is our top of our supported browser list.

Reply:
Yes -- as per commit message here: https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/2056/commits/144054bc891db4ce4e6a35c75f03f9eeeeaade53

Electron uses Chromium anyway, so any rendering and functionality should be on par with Chrome. This is mostly a convenience change, which prevents Cypress from refocusing the test window whenever a new test starts running.